hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/rsql/RSQLUtilityTest.java

```diff
@@ -99,6 +104,46 @@ TenantConfigurationManagementHolder tenantConfigurationManagementHolder() {
     private static final TenantConfigurationValue<String> TEST_POLLING_OVERDUE_TIME_INTERVAL = TenantConfigurationValue
             .<String> builder().value("00:07:37").build();
 
+    @Test
+    @Description("Testing throwing exception in case of not allowed RSQL key")
+    public void rsqlUnsupportedFieldExceptionTest(){
+        final String rsql1 = "wrongfield == abcd";
+        assertThatExceptionOfType(RSQLParameterUnsupportedFieldException.class)
```

Comment:
It is necessary to test the messages of `RSQLParameterUnsupportedFieldException` as well as `RSQLParserException`, probably in a separate method